src/main/java/com/google/alpollo/model/AnalysisInfo.java

```diff
@@ -15,7 +15,7 @@
 public class AnalysisInfo {
   @Id private Long id;
   /** All the info in this class is attached to a specific parent song. */
-  private final Song song;
+  @Index private Song song;
```

Comment:
Why did you remove `final`? Is this a requirement for using `@Index`?

Reply:
Yes, objectify don't save `final` fields